src/lib.rs

```diff
@@ -296,7 +296,7 @@ impl Config {
     ///             .file("src/foo.c")
     ///             .shared_flag(true)
     ///             .static_flag(true)
-    ///             .compile("libfoo.so");
```

Comment:
Can you review this line.  I'm really not sure if this is in scope.  IDK how .so passes that assert!